Sources/Nitrozen-SwiftUI/Elements/RadioButton/NitrozenRadio.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  NitrozenRadio.swift
+//  Example-Nitrozen-SwiftUI
+//
+//  Created by Rushang Prajapati on 05/04/23.
+//
+
+import Foundation
+import SwiftUI
+
+//MARK: NitrozenCheckbox
+public struct NitrozenRadio<Element>: View where Element: NitrozenElementRadioStringSelectableStyle & Hashable {
+    public enum Layout {
+        case horizontal, verticle
+    }
+    
```

Comment:
Add todo comment with your name 
//TODO: Rushang -- Move this layout type outside to use at multiple places

-> I will do these changes -- just add comment with your name and push